poncho/src/poncho_package_run

```diff
@@ -152,6 +152,12 @@ function cleanup {
     s=$((total_ms/1000))
 
     logmsg total runtime: $(printf "%d.%0.3d seconds" "${s}" "${ms}")
+    cpids=$(pgrep -P "$$")
+    for cpid in $cpids;
+    do
+        kill -SIGTERM $cpid
+        wait $cpid
```

Comment:
We want to send the kill to all  cpids and then `wait` without arguments for all of them.
In the current implementation, if one process is ignoring the SIGTERM, then we do not sent the signal to the rest of the processes. If at that point poncho_package_run gets another signal (e.g. from C-c), it will quit without sending the SIGTERM to all the processes.